util/topgen/validate.py

```diff
@@ -216,6 +216,9 @@
     'outgoing_alert': ['s', 'optional string to indicate alerts are routed externally to the named '
                             'group'],
     'incoming_interrupt': ['g', 'Parsed incoming interrupts (generated)'],
+    'racl_mapping': ['s', 'optional RACL mapping for that IP'],
+    'racl_mappings': ['g', 'dict of RACL mappings for all registers interfaces if there are more '
+                      'than one'],
```

Comment:
I'd probably list these in the other order, with something like:
- `racl_mappings`: dict that maps an interface to its associated RACL mapping.
- `racl_mapping`: A special case of racl_mappings. If specified, this is taken to represent a dict that associates all interfaces with the give mapping. It is an error to specify both this and racl_mappings.`